README.md

```diff
@@ -621,3 +808,36 @@ public class JpaCreate {
 
 We have learned how to create one-to-many associations and how to keep the
 relationships in sync in the program by creating a list of related entities.
+
+A project is associated with exactly one student, thus the `Project` class stores
```

Comment:
Wonderful conclusion / summary! I think this really wraps up the point of this lesson and what these annotations do.

Reply:
thanks!